internal/domain/tenant/repository.go

```diff
@@ -0,0 +1,10 @@
+package tenant
+
+import (
+	"context"
+)
+
+type Repository interface {
+	Create(ctx context.Context, tenant *Tenant) error
+	GetById(ctx context.Context, id string) (*Tenant, error)
+}
```

Comment:
We need to implement these repositories as well in `/internal/repository/postgres/tenant.go`

Reply:
done with this